crates/transaction-emitter-lib/src/emitter/mod.rs

```diff
@@ -41,7 +41,7 @@ use std::{
 use tokio::{runtime::Handle, task::JoinHandle, time};
 
 // Max is 100k TPS for a full day.
-const MAX_TXNS: u64 = 10_000_000_000;
+const MAX_TXNS: u64 = 100_000_000;
```

Comment:
this seems unnecessary change after updating the MAX_GAS_AMOUNT?

Reply:
Reverted.